controllers/helmrepository_controller.go

```diff
@@ -207,51 +204,63 @@ func (r *HelmRepositoryReconciler) reconcile(ctx context.Context, repository sou
 			return sourcev1.HelmRepositoryNotReady(repository, sourcev1.IndexationFailedReason, err.Error()), err
 		}
 	}
-	if err := chartRepo.DownloadIndex(); err != nil {
+	revision, err := chartRepo.CacheIndex()
```

Comment:
The SHA256 checksum value obtained is used to set the `Artifact.Revision` below.
The comment in the API spec mentions this to be helm index timestamp, which is no longer correct.

Reply:
The API spec for the Artifact mentions examples that can be used as a Revision (hence the "can be", and not "is").